### ISSUE TITLE
how does this work with third party storage vs first?

### ISSUE THREAD

**Issue by rugabunda** [wooque/site-bleacher#13]
just curious

**Comment by wooque** [comment]
What do you mean?

**Comment by rugabunda** [comment]
does it delete all third party files as well or only first party

**Comment by wooque** [comment]
@rugabunda  Yes, but it relies that third party also sets the cookie, because there is no extension api that will return all storages, only cookies, so when the cookie is deleted it will delete other browsing data. Thou on Firefox extension api for removing browsing data will remove all browsing data of certain types, not just for hostnames that are passed to that extension api
